repository/pom.xml

```diff
@@ -322,6 +322,12 @@
             <version>3.0.0-SNAPSHOT</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-validator</artifactId>
+            <version>4.3.0.Final</version>
```

Comment:
Got a CVE of severity medium here! please fix it. Can't let it to be a backlog
<img width="944" alt="image" src="https://github.com/atlanhq/atlas-metastore/assets/59254445/4723c6e3-1493-4030-8d1a-57b266cccb4c">
https://github.com/atlanhq/atlas-metastore/actions/runs/8890490131

Check how difficult it is to fix it.

Reply:
Basically dependency review is failing

Reply:
<img width="446" alt="image" src="https://github.com/atlanhq/atlas-metastore/assets/59254445/67cda48c-fa6e-4914-a03e-6f875eebaeed">
Just bump it to 4.3.2 and we will be fine here

Reply:
https://github.com/advisories/GHSA-845h-985r-jrqh

Reply:
Initially, I tried with the latest version for Java 8 supported https://central.sonatype.com/artifact/org.hibernate.validator/hibernate-validator/6.2.5.Final

This is not working as the repository/gorupId got changed from `org.hibernate` to `org.hibernate.validator` which is why the validator is not working (possible issue might be due to conflict in dependency). 

I have limited knowledge of how dependency works in Java. If you can help debug further, I am happy to jump on an call and solve this.